### PR TITLE
Now DataObject allows set/get package-private & protected

### PR DESCRIPTION
**Pull request by adaussy**
Signed-off-by: slinkydeveloper <francescoguard@gmail.com>
Signed-off-by: Arthur Daussy<arthur.daussy@gmail.com>

### CODE REVIEW

File: src/test/java/io/vertx/test/codegen/converter/TestDataObject.java [hunk 1]
```diff
@@ -11,1190 +11,1326 @@
...
-
-  public Map<String, NoConverterDataObject> getNotConvertibleDataObjectMap() {
-    return notConvertibleDataObjectMap;
-  }
-
-  public TestDataObject setNotConvertibleDataObjectMap(Map<String, NoConverterDataObject> notConvertibleDataObjectMap) {
-    this.notConvertibleDataObjectMap = notConvertibleDataObjectMap;
-    return this;
-  }
+	@Mapper
```
Comment (vietj) [inline src/test/java/io/vertx/test/codegen/converter/TestDataObject.java]: can you avoid reformating this code entirely and just add what is necessary ?

Comment (adaussy) [reply src/test/java/io/vertx/test/codegen/converter/TestDataObject.java]: It should be better now

File: pom.xml [hunk 2]
```diff
@@ -10,11 +10,11 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>vertx-codegen</artifactId>
-  <version>4.0.0-SNAPSHOT</version>
+  <version>3.8.4-SNAPSHOT</version>
```
Comment (vietj) [inline pom.xml]: why this change of version in master ?

**Comment by vietj** [comment]
please reopen a new PR
